tool/tctl/common/acl_command.go

```diff
@@ -234,20 +247,24 @@ func displayAccessLists(format string, accessLists ...*accesslist.AccessList) er
 }
 
 func displayAccessListsText(accessLists ...*accesslist.AccessList) error {
-	table := asciitable.MakeTable([]string{"ID", "Review Frequency", "Review Day Of Month", "Granted Roles", "Granted Traits"})
+	table := asciitable.MakeTable([]string{"ID", "Review Frequency", "Review Day Of Month", "Granted Roles", "Granted Traits", "Nested Owner Lists"})
```

Comment:
This feels a bit random: why do we show nested owner lists here when we don't display normal list owners or members?